### PR TITLE
changing rosa cli to 1.2.22 and truncating rosa version string

### PR DESCRIPTION
**Pull request by diana-sari**

### CODE REVIEW

File: plugins/modules/rosa_cluster.py [hunk 1]
```diff
@@ -229,8 +229,9 @@ def run_module():
     # check that rosa is minimum version
     # MIN_ROSA_VERSION
     rosa_version_cmd = [rosa, "version"]
+    # rosa_version_cmd = [rosa, "version", "|", "head", "-1"]
     rc, stdout, stderr = module.run_command(rosa_version_cmd)
-    rosa_version = stdout.rstrip()
+    rosa_version = stdout.rstrip()[:7]
```
Comment (paulczar) [inline plugins/modules/rosa_cluster.py]: I think this makes more sense `stdout.rstrip().split('\n')[0]`

File: plugins/modules/rosa_cluster.py [hunk 2]
```diff
@@ -229,8 +229,9 @@ def run_module():
     # check that rosa is minimum version
     # MIN_ROSA_VERSION
     rosa_version_cmd = [rosa, "version"]
+    # rosa_version_cmd = [rosa, "version", "|", "head", "-1"]
```
Comment (paulczar) [inline plugins/modules/rosa_cluster.py:232]: just delete this comment

File: plugins/modules/rosa_cluster_info.py [hunk 3]
```diff
@@ -90,8 +90,9 @@ def run_module():
     # check that rosa is minimum version
     # MIN_ROSA_VERSION
     rosa_version_cmd = [rosa, "version"]
+    # rosa_version_cmd = [rosa, "version", "|", "head", "-1"]
```
Comment (paulczar) [inline plugins/modules/rosa_cluster_info.py:93]: see above

File: plugins/modules/rosa_cluster_info.py [hunk 4]
```diff
@@ -90,8 +90,9 @@ def run_module():
...
     rc, stdout, stderr = module.run_command(rosa_version_cmd)
-    rosa_version = stdout.rstrip()
+    rosa_version = stdout.rstrip()[:7]
```
Comment (paulczar) [inline plugins/modules/rosa_cluster_info.py]: see above

**Review by paulczar — CHANGES_REQUESTED** [review]

File: plugins/modules/rosa_cluster.py (hunk 1 above)
Comment (diana-sari) [reply plugins/modules/rosa_cluster.py]: updated

File: plugins/modules/rosa_cluster.py (hunk 2 above)
Comment (diana-sari) [reply plugins/modules/rosa_cluster.py:232]: done

File: plugins/modules/rosa_cluster_info.py (hunk 3 above)
Comment (diana-sari) [reply plugins/modules/rosa_cluster_info.py:93]: done

**Review by paulczar — APPROVED** [review]

**Pull request merged**